Automattic/WP-Job-Manager
# Add additional protection for people using `[jobs]` in a very custom way

#### Changes proposed in this Pull Request:

* Fixes a very minor/edge case issue restoring the form from when it didn't exist. The only real way to produce this is have the page be set to `[jobs show_filters="false"]`, load the page, and secretly change it to `[jobs]`. 
* Fixes an issue where themes might either use `do_shortcode` or build the `[jobs]` shortcode and use `ajax-filters.js` to load the listing (often in archive pages).

#### Testing instructions:

* For first issue, do those changes as described.
* For the second issue, try Jobify's archive page for a job manager taxonomy (tags, categories, etc).  The state should be stored in a session variable (On Chrome, in Inspector go to Application > Session Storage > _domain_) that doesn't include (`undefined`). For example:

Good:  `job_listing_http://wpjm.docker/job-tag/cool/_0`

Bad: `job_listing_undefined_0`
